CMakeLists.txt

```diff
@@ -185,12 +185,14 @@ if(NOT COMMAND find_host_program)
 endif()
 
 # Tests require Python3
-find_host_package(PythonInterp 3 REQUIRED)
+if(NOT SPIRV_SKIP_TESTS)
+  find_host_package(PythonInterp 3 REQUIRED)
+endif()
 
 # Check for symbol exports on Linux.
 # At the moment, this check will fail on the OSX build machines for the Android NDK.
 # It appears they don't have objdump.
-if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+if(NOT SPIRV_SKIP_TESTS AND "${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
I'm not sure what you are trying to accomplish this this change.  If tests are being skipped, then the macro `spvtools_check_symbol_exports` will do nothing in both cases.

Reply:
You're right, I just didn't look at that close enough. I think at some point I saw an error that prompted me to make that change, but it may have been caused by something unrelated.